load-gen/src/main/java/io/smartcat/data/loader/api/Terminateable.java

```diff
@@ -0,0 +1,13 @@
+package io.smartcat.data.loader.api;
+
+/**
+ * Has ability to terminate itself.
+ */
+public interface Terminateable {
```

Comment:
As LoadGenerator is the entry point and it creates all components then the cleanup is his responsibility